src/components/cardsList/CardsList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './CardsList.css';
+import { ICard } from '../../interfaces';
+import data from '../../data/db.json';
+import Card from '../../components/Card/Card';
```

Comment:
Import path casing should match exactly.
'../../components/card/Card' folder 'card' should be in lower case.

Reply:
It's bug of git. I fixed, commited, but in repo changes didn't appear.